forms/OptionsetField.php

```diff
@@ -102,4 +102,19 @@ public function validate($validator) {
 	public function ExtraOptions() {
 		return new ArrayList();
 	}
+	
+	public function getAttributes() {
+		$attributes = parent::getAttributes();
+		unset($attributes['name']);
+		if(isset($attributes['required'])) {
+			unset($attributes['required']);
+		}
+		if(isset($attributes['role'])) {
+			unset($attributes['role']);
+		}
+		if(isset($attributes['aria-required'])) {
```

Comment:
could we could do something smart, like checking if there is a fieldset as a parent and then set the aria-required on it?